ios/Sources/Model/Image.swift

```diff
@@ -0,0 +1,23 @@
+import DroidKaigiMPP
+
+public struct Image: Equatable {
+    public var largeURLString: String
+    public var smallURLString: String
+    public var standardURLString: String
+
+    public init(
+        largeURLString: String,
+        smallURLString: String,
+        standardURLString: String
+    ) {
+        self.largeURLString = largeURLString
+        self.smallURLString = smallURLString
+        self.standardURLString = standardURLString
+    }
+
+    public init(from model: DroidKaigiMPP.Image) {
+        self.largeURLString = model.largeUrl
+        self.smallURLString = model.smallUrl
+        self.standardURLString = model.standardUrl
+    }
```

Comment:
Better creating enum and function to take out? which is like
```
public func getByLang(lang: Lang) -> String
```

Reply:
I think this is not needed. If I do it, I will have to define `ImageSize` enum and it's redundancy.